src/lib/firsts.rs

```diff
@@ -37,7 +37,7 @@ use cfgrammar::{Grammar, NTIdx, Symbol, TIdx};
 use cfgrammar::yacc::YaccGrammar;
 
 // Firsts stores all the first sets for a given grammar. For example, given this code and grammar:
```

Comment:
I agree with you that the comments was a bit confusing, so I've tried fixing it.

Triple quotes aren't needed for multiline strings in Rust though.